docs/api-conventions.md

```diff
@@ -238,8 +238,6 @@ The following HTTP status codes may be returned by the API.
 
 * `200 StatusOK`
   * Indicates that the request completed successfully.
-* `201 StatusCreated`
```

Comment:
Why are you removing 201?

We do return 201:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/test/integration/auth_test.go#L201

#2031